test/complexity/measures/statistical_complexity.jl

```diff
@@ -0,0 +1,35 @@
+using ComplexityMeasures, Distances
```

Comment:
The tests fail because `Distances` is not part of the `test/Project.toml` file. 

Please add an entropy for Distances under ["deps"], either manually by copying it from the main `Project.toml` file, or by activating the `ComplexityMeasures.jl/test` folder and then running `using Pkg; Pkg.add("Distances")`.